docs/DEPLOYMENT.md

```diff
@@ -381,50 +378,10 @@ For `OpenShift`:-
 
 Open the URL in a browser.
 
-Note: In UI you won't be able see the resources on homepage as there are not any in the db but you should see Categories on the left side.
+Note: In UI you should able to see the resources on homepage and Categories on the left side.
 
 Update your GitHub OAuth created with the UI route in place of `Homepage URL` and `Authorization callback URL`.
 
-## Add resources in DB
```

Comment:
I think we should not delete this, rather change the heading because if someone wants to update the catalog he needs to refresh the catalog